### PR TITLE
Conversions: "oz" = "oz fl" if "ml" contained in query

### PR DESCRIPTION
**Pull request by MrChrisW**
#1125

### CODE REVIEW

File: lib/DDG/Goodie/Conversions.pm [hunk 1]
```diff
@@ -76,13 +76,20 @@ handle query_lc => sub {
 
     # hack support for "degrees" prefix on temperatures
     $_ =~ s/ degrees (celsius|fahrenheit)/ $1/;
+    
+    # hack - convert "oz" to "fl oz" if "ml" contained in query
+    if($_ =~ /\boz\b\s+[A-Z]{1,4}\s+\bml\b|\bml\b\s+[A-Z]{1,4}\s+\boz\b/i) {
```
Comment (MrChrisW) [inline lib/DDG/Goodie/Conversions.pm]: @mintsoft Any suggestions on simplifying this? :smiley:

Comment (mintsoft) [inline lib/DDG/Goodie/Conversions.pm]: Can simplify by worrying a little less about format with something like: `s/ oz / fl oz / if($_ =~ /ml/ && !~ / fl oz /)` (untested!)

Comment (MrChrisW) [inline lib/DDG/Goodie/Conversions.pm]: @mintsoft thanks for the tip! this should [do it](https://github.com/duckduckgo/zeroclickinfo-goodies/pull/1127/files#diff-e40dae3e0dc357950d79ca5c2e964b3eR81)

**Comment by mintsoft** [comment]
Cool LGTM!

**Pull request merged**